app/models/node.rb

```diff
@@ -750,6 +750,19 @@ def add_tag(tagname, user)
                                  uid: user.uid,
                                  date: DateTime.now.to_i,
                                  nid: id)
+
+          # Adding lat/lon values into node table
+          if tag.valid?
+            if tag.name.split(':')[0] == 'lat'
+              tagvalue = tag.name.split(':')[1]
+              self.update_attribute(:latitude, tagvalue)
```

Comment:
OK this looks good but won't we need to change the type? to a decimal? 

Perhaps let's write a unit test to try this out - doing `node.add_tag('lat:10.0002', users(:bob))` and seeing if it adds the correct value and precision. If it fails due to the type not being correct, we can fix it after -- TDD!